.github/workflows/deploy.yml

```diff
@@ -49,21 +49,59 @@ jobs:
         name: cargo-c-windows-${{ matrix.conf }}-binaries
         path: cargo-c-windows-${{ matrix.conf }}.zip
 
+  linux-binaries:
 
-  unix-binaries:
+    runs-on: ubuntu-latest
 
-    strategy:
-      matrix:
-        conf:
-         - linux
-         - macos
-        include:
-         - conf: linux
-           os: ubuntu-latest
-         - conf: macos
-           os: macos-latest
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Install musl-tools
+      run: |
+        sudo apt-get install musl-tools
+
+    - name: Install Rust stable and musl target
+      uses: actions-rs/toolchain@v1
+      with:
+        profile: minimal
+        toolchain: stable
+        target: x86_64-unknown-linux-musl
+        override: true
+
+    - name: Build openssl for musl
+      run: |
+         OPENSSL_BIN=target/x86_64-unknown-linux-musl/release/build
+         git clone --depth 1 https://github.com/sfackler/rust-openssl
+         cd rust-openssl/openssl-sys
+         cargo build --target x86_64-unknown-linux-musl \
+                     --release \
+                     --features=vendored
+         mv ../$OPENSSL_BIN/openssl-sys-*/out/openssl-build/install/* \
+            $GITHUB_WORKSPACE
+
+    - name: Build cargo-c
+      run: |
+        export OPENSSL_LIB_DIR=$GITHUB_WORKSPACE/lib
+        export OPENSSL_INCLUDE_DIR=$GITHUB_WORKSPACE/include
+        cargo build --target x86_64-unknown-linux-musl --release
+
+    - name: Create zip
+      run: |
+        cd target/x86_64-unknown-linux-musl/release
+        strip cargo-cbuild cargo-cinstall
+        zip $GITHUB_WORKSPACE/cargo-c-linux.zip \
```

Comment:
make a tar please :)

Reply:
Is there a particular reason not to use `zip`?

Reply:
tar is better understood on *nix systems and sometimes they do not come with any un-zip programs by default

Reply:
Seems a good reason, thank you @1480c1 :)